README.adoc

```diff
@@ -6,7 +6,7 @@ toc::[]
 
 == Introduction
 
-Kiali provides answers to the question _What are the microservices in my Istio service mesh doing?_
+Kiali provides answers to the questions: _What are the microservices in my Istio service mesh, and how are they connected?_
```

Comment:
Maybe "What microservices exist in my Istio service mesh" instead of "what are ....", otherwise I feel it misses a verb :)

Reply:
I would choose either "What microservices are part of my service mesh and how are they connected."  or "Which microservices exist in my service mesh and how are they connected."

Reply:
@jotak have you addressed @JStickler comments?

Reply:
done